blueocean-rest/src/main/java/io/jenkins/blueocean/rest/factory/BlueIssueFactory.java

```diff
@@ -1,26 +1,21 @@
 package io.jenkins.blueocean.rest.factory;
 
 
-import com.google.common.collect.Lists;
-import com.google.common.collect.Sets;
 import hudson.ExtensionList;
 import hudson.ExtensionPoint;
 import hudson.model.Job;
 import hudson.scm.ChangeLogSet;
 import io.jenkins.blueocean.rest.model.BlueIssue;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.Collection;
-import java.util.LinkedHashMap;
 import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Set;
-import java.util.logging.Level;
-import java.util.logging.Logger;
 
 public abstract class BlueIssueFactory implements ExtensionPoint {
 
 
-    private static final Logger LOGGER = Logger.getLogger(BlueIssueFactory.class.getName());
+    private static final Logger LOGGER = LoggerFactory.getLogger( BlueIssueFactory.class.getName());
```

Comment:
Is this required? 
```suggestion
    private static final Logger LOGGER = LoggerFactory.getLogger(BlueIssueFactory.class.getName());
```

Reply:
yes to have some logging everywhere. as the code is a mix